### ISSUE TITLE
About the video diffusion model used in this paper

### ISSUE THREAD

**Issue by tokki8682** [minsu1206/ExploreGS#3]
Thank you for releasing this impressive work.

In the paper, the issue of blurry pixels is mentioned in Section 6 (Conclusion) and also in Supplementary Material E (Future Works). This seems to stem from the limited resolution of the generated images produced by the video diffusion model, which leads to the loss of fine details.

Currently, as far as I understand, a 320×512 resolution video diffusion model is used. But a higher-resolution DynamiCrafter model with 576×1024 resolution has been released.

I would like to ask:
Are there any plans to replace or extend the current diffusion model with this higher-resolution DynamiCrafter version (or other high-resolution video diffusion models) to mitigate the blurry pixel issue?

**Comment by minsu1206** [comment]
Thank you for your interest in our work, and apologies for the late response.

Due to limited GPU resources, we do not currently plan to replace the model with a higher-resolution diffusion model.

Still, we believe it would likely reduce pixel-level blurriness, since in our experiments, upsampling the diffusion output to the original image resolution adds blur.

Thanks.

**Comment by tokki8682** [comment]
Thanks for your quick reply. I understand.

As a follow-up question:
Is it possible to release the dataset used to train video diffusion model?
If full release is not possible, a partial release would also be greatly appreciated.

( I would like to fine-tune a higher-resolution model on my own and compare the results. )

Also, regarding the 3DGS_enhancer_extra repository: I noticed that while Adapter and ControlNet modules are implemented, they appear to be unused in config_mv_run1_relpose_dl3dv_3k_re10k_1k.yaml (which I assume was used to train enhancer.ckpt). May I ask the reason why they were not utilized in the final configuration?

**Comment by minsu1206** [comment]
Thank you for your understanding.

1. Dataset release
Unfortunately, we cannot afford to release whole dataset. Sorry.
It's over 3TB, and releasing and maintaining it costs lot to us.
Instead, we'll upload the codes we used to construct the dataset and examples.
Actually, these codes are largely same as original 3DGS repo, only except at sampling input sequences and target frames to be enhanced. There could be better strategy for sampling them than ours.

2. No use of RE10K
That's also good point. 
According to our observations, reconstructed 3D scenes in RE10K following original 3DGS (even we adapted original train/test set sampling!) and using provided camera poses show noisy reconstruction results. 
I still don't have no idea about it. My opinion (might be wrong , not precise ... ) : most of RE10K scenes are forward-facing scenes. 3DGS seems weak at forward-facing scenes.
To address forward-facing scenes, NeRF uses NDC but 3DGS doesn't, so it leads to 3DGS suffer from geometry uncertainty.

Therefore, they often suffers from floaters or blurry rendering results. This phenomenon was sometimes too severe to make learning more difficult. (In contrast, reconstructed DL3DV10K scenes shows affordable artifacts to be enhanced by diffusion model.)

As a result, we found that models trained at only DL3DV set shows better performance.

Feel free to ask any question.
Thank you for your interest again.

**Comment by tokki8682** [comment]
Thank you again for your detailed explanations.

As a final clarification, may I please confirm the following points?
➀Is my understanding correct that the currently released checkpoint, enhancer.ckpt, was trained using the configurations folder "3DGS_Enhancer_Extra/configs/training_512_mv/valid_experiments/" ?

②In the 3DGS_Enhancer_Extra repository, there are ControlNet (models/control.py) and Adapter (networks/add_on/adapter.py) modules are implemented, which are not present in the original DynamiCrafter repository.
Could you please confirm whether **enhancer.ckpt** was trained **without using these ControlNet and Adapter modules**?

**Comment by minsu1206** [comment]
(1) Oh, that's my mistake. Thank you for pointing this. As mentioned earlier, we didn't use RE10K. `configs/training_512_mv/config_mv_run1_relpose_3k.yaml` would be the final configuration we used.

(2) We didn't use those additional modules. (If my memory serves me correctly.) There were legacy experiments we tried - incorporating some priors for diffusion enhancer. (Not working well, so we didnt' use anymore.) 
The main difference you should focus is how we did conditioning frames. `https://github.com/minsu1206/3DGS_Enhancer_Extra/blob/main/lvdm/models/ddpm3d.py#L1344`
You can easily find that original DynamiCrafter and ours choose different conditioning strategy.

**Comment by tokki8682** [comment]
Thank you so much for taking the time to answer my questions in such detail.
I really appreciate your thoughtful and informative responses.
